### PR TITLE
Switched to use npmignore to expose scss files

### PR DESCRIPTION
**Pull request by heff**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Exclude everything but the contents of the dist directory.
 **/*
 !dist/**
+!src/css/**
```
Comment (gkatsev) [inline .npmignore]: do we want to ignore the font folder?
If so, add `src/css/font/**` on the following line.

Comment (heff) [inline .npmignore]: I don't think we do? Seems like that's a key part of the CSS, but I'll leave that up to Matt. I don't think it's hurting anything to include them.

Comment (mmcc) [inline .npmignore]: We do want the font folder as well. I, for instance, immediately copied over the font folder via grunt-contrib-copy. Dist should contain everything necessary to self-host a player.

On a different note (that shouldn't he handled here), we should consider renaming this folder `scss`.

**Comment by mmcc** [comment]
lgtm

**Pull request merged**